database/src/tables.rs

```diff
@@ -74,16 +77,71 @@ macro_rules! tables {
 }
 
 //---------------------------------------------------------------------------------------------------- Tables
+// Notes:
+// - Keep this sorted A-Z (by table name)
+// - Tables are defined in plural to avoid name conflicts with types
+// - If adding/changing a table, also edit the tests in `src/backend/tests.rs`
+//   and edit `Env::open` to make sure it creates the table
 tables! {
-    /// Test documentation.
-    TestTable,
-    true,
-    i64 => TestType,
-
-    /// Test documentation 2.
-    TestTable2,
-    true,
-    u8 => TestType2,
+    /// TODO
+    BlockBlobs,
+    BlockHeight => BlockBlob,
+
+    /// TODO
+    BlockHeights,
+    BlockHash => BlockHeight,
+
+    /// TODO
+    BlockInfoV1s,
+    BlockHeight => BlockInfoV1,
+
+    /// TODO
+    BlockInfoV2s,
+    BlockHeight => BlockInfoV2,
+
+    /// TODO
+    BlockInfoV3s,
+    BlockHeight => BlockInfoV3,
+
+    /// TODO
+    KeyImages,
+    KeyImage => (),
+
+    /// TODO
+    NumOutputs,
+    Amount => AmountIndex,
+
+    /// TODO
+    PrunedTxBlobs,
+    TxId => PrunedBlob,
+
+    /// TODO
+    Outputs,
+    Amount => Output, // FIXME: `Amount | AmountIndex` key
```

Comment:
This is `u64 | u64` so the type key is `u64` but what should the alias name be?

Reply:
`PreRCTOutputID`?

 By `u64 | u64` I meant a byte concatenation so a [u8; 16] but the key should be `(u64, u64)` to Rust

Reply:
Ah, I see. It will have to be a named struct since `Pod` isn't implemented on tuples.